python-with-pandas/3.6-alpine/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.6-alpine3.9
+
+LABEL maintainer=pcrespov
+
+RUN apk add --no-cache --virtual .BUILD-DEPS \
+        build-base \
+        git \
+        gcc \
+        wget \
+        libc-dev \
+        python-dev \
+        musl-dev \
+        libffi-dev \
+        postgresql-dev
```

Comment:
We need some way to remove all that dev stuff from the final image as this makes them unnecessary big.
we only need the runtime libraries in the end.

What about having something at the end like:
```
FROM base as alpine-pandas
RUN apk add --no-cache \
    RUNTIME_LIBRARIES
COPY from=base pip-pandas libraries
```

Reply:
yes, the dev libraries are removed at the end. notice that the ``--virtual`` in the ``apk add`` CL